python/amico/core.py

```diff
@@ -51,8 +52,9 @@ def __init__( self, study_path, subject ) :
 
         self.set_config('peaks_filename', None)
         self.set_config('doNormalizeSignal', True)
+        self.set_config('doKeepb0Intact', False)  # does change b0 images in the predicted signal
         self.set_config('doComputeNRMSE', False)
-
+        self.set_config('doSaveCorrectedDWI', False)
```

Comment:
I saw some modification in `core.py` as well:

- [Lines 272-274](https://github.com/mdesco/AMICO/blob/0625d984790cd79b57efac3c471a79aa70ff3b68/python/amico/core.py#L272-L274)
- [Line 313](https://github.com/mdesco/AMICO/blob/0625d984790cd79b57efac3c471a79aa70ff3b68/python/amico/core.py#L313)

The parameters `d_isos` might not be present in others models, e.g. NODDI. Or perhaps I am messing up while looking at the diff file (my first pull request..)